model/error/_meta/fields.yml

```
@@ -4,7 +4,7 @@
   fields:
     - name: view errors
       type: keyword
-      script: "doc['context.service.name'].value"
+      script: "doc['service.name'].size() > 0 ? doc['service.name'].value : doc['context.service.name'].size() > 0 doc['context.service.name'].value : null"
```

Doesn't this have a typo in the second ternary?

Shouldn't it be (notice the added `?`):
```suggestion
      script: "doc['service.name'].size() > 0 ? doc['service.name'].value : doc['context.service.name'].size() > 0 ? doc['context.service.name'].value : null"
```